cmplibrary/src/test/java/com/sourcepoint/cmplibrary/data/ServiceImplTest.kt

```diff
@@ -461,4 +461,170 @@ class ServiceImplTest {
         verify(exactly = 2) { ncMock.postPvData(any()) }
         verify(atLeast = 2) { cm.ccpaConsentStatus = any() }
     }
+
+    @Test
```

Comment:
We need to focus on UI tests, even if this test gives you a green check, the graceful degradation feature doesn’t work properly